UnitTests/BraintreePayPalTests/BTPayPalClient_Tests.swift

```diff
@@ -152,21 +152,20 @@ class BTPayPalClient_Tests: XCTestCase {
         ])
 
         mockWebAuthenticationSession.cannedResponseURL = URL(string: "https://onetouch/v1/success")
-        let expectation = expectation(description: "Edit vault completion")
-        clientTokenPayPalClient.edit(editRequest) { result, error in
-            XCTAssertNil(error)
-            XCTAssertNotNil(result?.riskCorrelationID)
-            expectation.fulfill()
+
+        do {
+            let _ = try await clientTokenPayPalClient.edit(request)
+        } catch {
+            // no-op
         }
```

Comment:
I think you can simplify this by doing:

```suggestion
        let _ = try? await clientTokenPayPalClient.edit(request)
```

Reply:
> You use try? to handle an error by converting it to an optional value. If an error is thrown while evaluating the try? expression, the value of the expression is nil. 

[Apple docs](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/errorhandling/#Converting-Errors-to-Optional-Values)

Reply:
❤️ nice, updated them all here: https://github.com/braintree/braintree_ios/pull/1421/commits/2181f9de4fd1fd4ee57a236e64ddb4ea05fca2d9